src/main/java/com/duke/util/Parser.java

```diff
@@ -0,0 +1,57 @@
+package com.duke.util;
+
+import com.duke.command.*;
+import com.duke.exception.DukeException;
+import com.duke.task.Deadline;
+import com.duke.task.Event;
+import com.duke.task.Todo;
+
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+
+    /**
+     * Parse the input by the user.
+     * @param input message input by the user.
+     * @return Command that can execute according to the user's input.
+     * @throws DukeException if there is invalid argument.
+     */
+    public static Command parse(String input) throws DukeException {
```

Comment:
This method is quite long, maybe you could refactor it? The `parse()` method name could be more descriptive as well.